README.md

```diff
@@ -1,12 +1,17 @@
-# Microverse_HTML_CSS_JS-Template
+# Yemen National Youth Party
+
```

Comment:
Great job @GeekyHacks 🤝

- [ ] As specified in the project requirements, Kindly ensure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).